algorithms/integration/kapton_2019_correction.py

```diff
@@ -323,21 +323,9 @@ def _check_int_edge_pts(int_edge_pts):
         # Sort out the edge points and the int_edge_points which are on the same side
         kapton_edge_1 = (col(int_edge_pts[0]) - col(int_edge_pts[1])).normalize()
         kapton_edge_2 = (col(int_edge_pts[2]) - col(int_edge_pts[3])).normalize()
-        min_loss_func = -999.9
```

Comment:
I think the original intent of this now deleted logic was to reorder the indices of the detector edge points from counterclockwise to clockwise (starting from origin) based on how well the edges associated with kapton absorption max and min align with the sides of the detector. For example, if the kapton absortion edges are more parallel to the top/bottom sides of the detector, then the detector edge point are re-ordered to reflect that. 

This logic was deleted because this re-ordering causes a divide by zero error on line 329

Reply:
I didn't attempt to re-implement a working version of this logic because I'm not certain if what the ultimate outcome the original logic sought to obtain or of its validity.